megamek/src/megamek/common/AmmoType.java

```diff
@@ -333,9 +333,11 @@
     
     // More SRM+LRM Munitions types
     public static final long M_MINE_CLEARANCE = 1l << 61;
+    
+    public static final long M_FAE = 1l << 63;
   
   
-    // Numbers 62-63 are used for Nuclear munitions, above 
```

Comment:
It would be helpful to add a comment here that 62 is already in use, as is done elsewhere that the numbers are out of order.